requirements-doc.txt

```diff
@@ -81,6 +81,7 @@ sphinxcontrib-jquery==4.1
 sphinxcontrib-jsmath==1.0.1
 sphinxcontrib-qthelp==1.0.7
 sphinxcontrib-serializinghtml==1.1.10
+sphinx-toolbox==3.5.0
```

Comment:
Since there are some fixes to be done anyway, please also add `sphinx-toolbox` [here](https://github.com/canonical/rockcraft/blob/main/setup.cfg#L47) (we use those deps when freezing the requirements).

Reply:
noted. done in cb6e68a